GliaWidgets/Sources/ViewModel/Chat/Data/Gva.swift

```diff
@@ -13,27 +13,44 @@ struct GvaResponseText: Decodable, Equatable {
         type = try container.decode(GvaCardType.self, forKey: .type)
         let contentString = try container.decode(String.self, forKey: .content)
         let modifiedString = contentString.replacingOccurrences(of: "\n", with: "</br>")
-        content = modifiedString.htmlToAttributedString ?? NSAttributedString(string: "")
+        content = modifiedString.htmlToAttributedString ?? NSMutableAttributedString(string: "")
     }
 }
 
-struct GvaButton: Decodable, Equatable {
+struct GvaButton: Codable, Equatable {
```

Comment:
Why do you need `Encodable`?

Reply:
You are right, this is not needed anymore. Since Metadata makes it impossible to encode it in a straightforward way, this approach was abandoned. I will remove it

Reply:
fixed [e84326b](https://github.com/salemove/ios-sdk-widgets/pull/705/commits/e84326b7cd81b80cecc5ec5b13b6881ff6057432)